arches/big_core.yaml

```diff
@@ -17,42 +17,68 @@ top.cpu.core0:
     l1_size_kb: 64
 
 top.cpu.core0.extension.core_extensions:
-  execution_topology:
-  [["alu", "6"],
-   ["fpu", "2"],
-   ["br",  "2"]]
-
+  # this sets the pipe targets for each execution unit
+  # you can set a multiple or just one:
+  # ["int", "div"] would mean this execution pipe can accept
+  # targets of: "int" and "div"
+  pipelines:
+  [
+    ["int"], # exe0
+    ["int", "div"], # exe1
+    ["int", "mul"], # exe2
+    ["int", "mul", "i2f", "cmov"], # exe3
+    ["int"], # exe4
+    ["int"], # exe5
+    ["float", "faddsub", "fmac"], # exe6
+    ["float", "f2i"], # exe7
+    ["br"], # exe8
+    ["br"] # exe9
+  ]
+  # this is used to set how many units per queue
+  # ["0", "1"] means iq0 has exe0 and exe1, so it's inclusive
+  # if you want just one execution unit to issue queue you can do:
+  # ["0"] which would result in iq0 -> exe0
+  # *note if you change the number of issue queues, 
+  # you need to add it to latency matrix below
+  issue_queue_to_pipe_map:
+  [ 
+    ["0", "1"], # iq0 -> exe0, exe1
+    ["2", "3"], # iq1 -> exe2, exe3
+    ["4", "5"], # iq2
+    ["6", "7"], # iq3
+    ["8", "9"]  # iq4
```

Comment:
Might want to finish the documentation here.

Reply:
Fixed in [19ed4b7](https://github.com/riscv-software-src/riscv-perf-model/pull/140/commits/19ed4b7911a0f115a2f9e2c2c19e5ad3324bbede)